src/graphnet/data/sqlite_dataconverter.py

```diff
@@ -130,14 +132,12 @@ def _parallel_extraction(self, settings):
         dataframes_big = OrderedDict([
             (key, pd.DataFrame()) for key in self._table_names
         ])
-
```

Comment:
I would suggest to avoid pure whitespace/newline changes in cases when these are (seemingly?) not important to improve legibility, or similar. See also changes at (old) L64, L140, L164, L310.